### DISCUSSION TITLE
Skidware - Fake DDoS ☠

### DISCUSSION THREAD

**Discussion by WiiZARDD** [blackhatethicalhacking/ddos#18] [General]
sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
            sock.connect((ip, port))
            sent += 1
            print(f"SYN Packets sent: {sent} to target: {ip}")

"Packets sent" 🤣 You've just established a TCP connection to specified port using your own network... SKID 😂😂👉

**Comment by blackhatethicalhacking** [comment]
Congrats for explaining how opening a TCP socket work in the snippet in Python! BTW, this is what it is intended to do when you select a SYN Flood attack. A very constructive discussion especially when you know the difference between a script and this.

FAKE DDoS = FAKE Comment

**Reply by WiiZARDD** [reply]
> Congrats for explaining how opening a TCP socket work in the snippet in Python! BTW, this is what it is intended to do when you select a SYN Flood attack. A very constructive discussion especially when you know the difference between a script and this.
> 
> 
> 
> FAKE DDoS = FAKE Comment

Closing my discussion topic 🤡
We've seen these skid tools from time to time again you're fooling nobody but yourself - Go create an IoT something profitable maybe something that can (actually) do something it's hilarious you try to defeat my point, you're harming nobody but yourself using this sKript. 🤡🥸

**Comment by WiiZARDD** [comment]
> Congrats for explaining how opening a TCP socket work in the snippet in Python! BTW, this is what it is intended to do when you select a SYN Flood attack. A very constructive discussion especially when you know the difference between a script and this.
> 
> 
> 
> FAKE DDoS = FAKE Comment

"SYN Packets sent" Some technical bs that is not true at all 🤡🔨 

More like "TCP Connection made" 😂😂🫵

**Comment by blackhatethicalhacking** [comment]
Profitable like VAC Bypass lol aight bro i thought your a pr0sh1t poster, u got time with words, i can see ur work, pr0n

**Reply by WiiZARDD** [reply]
> time

That's a fork you nerd open-source repository for a forum thread discussion, irrelevant anyways. 

I am good with my words I mind as well be a poet, I am just as good with reading too. 

Good introductory repository for simple networking related needs though this lacks proper "DDoS" functionality, such as an IoT or simple qBot for example, therefore this is nothing more than a personal tool - Time and time again I see repositories as such falsely stating the obvious, please revise, you are misleading skids.
